vico/core/src/main/java/com/patrykandpatrick/vico/core/cartesian/layer/LineCartesianLayer.kt

```diff
@@ -348,9 +348,11 @@ public open class LineCartesianLayer(
         line.draw(context, linePath, lineFillCanvas, verticalAxisPosition)
         canvas.drawBitmap(lineFillBitmap, 0f, 0f, null)
 
-        forEachPointInBounds(series, drawingStart, pointInfoMap) { entry, x, y, _, _ ->
+        // ----------------------- WAHOO START --------------------------
+        /*forEachPointInBounds(series, drawingStart, pointInfoMap) { entry, x, y, _, _ ->
           updateMarkerTargets(entry, x, y, lineFillBitmap)
```

Comment:
unused code solely for updating markers that we don't use

saves about .25ms per graph point